lib/src/DynamicProto.ts

```diff
@@ -18,6 +18,22 @@ interface DynamicGlobalSettings {
 
 
 
+/**
+ * Helper to check if we're running in a server-side rendering environment
+ * like Node.js or Cloudflare Workers
+ * @ignore
+ */
+function _isServerSideRender(): boolean {
+    // Check for common server-side environments
+    // 1. Missing window or document (Node.js, some SSR frameworks)
+    // 2. Cloudflare Worker specific environment
+    return (!hasWindow() || 
```

Comment:
use getWindow() and getDocument() instead

Reply:
use getWindow() and getDocument() instead of hasXXXX as the has functions just call the get versions anyway